<launch>
  <node name="robot_movement" pkg="final_project" type="robot_movement.py" output="screen" />
</launch>
